src/screens/AboutPage.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
This file seems to be a duplicate and does not seem to be adhering to the folder structure we had discussed